### ISSUE TITLE
Undo ThisIsWin11 changes!

### ISSUE THREAD

**Issue by Hakim-Allaoui** [builtbybel/ThisIsWin11#3]
Hello,
I would like to know if it is possible to undo ThisIsWin11 to revert the changes made to File Explorer, I want to disable Win 10 File Explorer, and keep the win 11 look

**Comment by Hakim-Allaoui** [comment]
I fix it by deleting the key named "{e2bf9676-5f8f-435c-97eb-11607a5bedf7}" located at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell Extensions\Blocked

**Comment by Trimonde** [comment]
I am INCREDIBLY PISSED OFF.
I worked so hard for two days to install Windows 11 on my unsupported PC . ThisIsWin11 messed me all up! I just want to my pc back to how it was two hours ago!! There seems to no longer be a way of going back to recovery points on Win11. What these people don't realize when they make these "user friendly" programs, is that us non programming-savvy users, who nonetheless understand what it is to have a lighter computer less chained into the system and so forth, will trustingly press buttons, and then when all of the sudden our useful useful icons disappear, and our interface changes way more than we wanted it to, we have no way of reverting things! I tried going back to Win11 defaults (on the program), and it just made things worse... trying to install some Win10 Microsoft package that made me panic and just want to shut every thing down! All my thumbnails are now generic squares. My unwanted color personalizations changes that I managed to revert, now keep going back to whatever ThisIsWin11 did to my pc!. The world doesn't belong to people who know how to navigate inside the brains of computers, so stop leading those of us who are not all gong-ho about how computers work IN THERE! If you are going to make something available to the general public that potentially can cripple your computer, at the least have the sensibility of working in a way of undoing things and putting that computer back to the way it was before!
